eeg_models/train_braininvaders.py

```diff
@@ -0,0 +1,129 @@
+from importlib import reload
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+
+import transforms
+from datasets import BrainInvadersDataset
+from eegnet import EegNet
+
+
+def main():
+    """Prepare dataset and dataloaders"""
+    raw_dataset = BrainInvadersDataset().data
```

Comment:
If we write code in the manner that it don't load all the data at once, would this work?

Reply:
I fixed it